examples/express-ts/src/models/interfaces.ts

```diff
@@ -0,0 +1,42 @@
+import {Model} from 'objection';
```

Comment:
If you add this, please wrap it with a big jsdoc disclaimer that it's completely optional. I think people will most likely use the express-ts template as an example of best-practices, and I've found this approach to be decidedly non-DRY while not adding additional type safety.

I'd remove it, personally.

Reply:
I'm not sure what you mean by this. I don't see how you can reference eager relation properties in a type-safe way without using these interfaces. If you try to do it without interfaces, TS barfs on circular dependency. Unless you can show me another way.

Yeah, interfaces are always non-DRY.

Reply:
Ah scheisse. I just stripped out the interfaces and now it's working despite the cyclic dependency. I don't know what changed. I'll push it.

Reply:
I've had a steep learning curve with Typescript. It has to be the most complex language I've ever tackled. Even C++ was more predictable.